alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt

```diff
@@ -582,6 +582,116 @@ class MonitorRunnerIT : AlertingRestTestCase() {
                 actionResults2[actionThrottleEnabled.id]!!.lastExecutionTime)
     }
 
+    fun `test monitor HttpInput with non JSON response `() {
+        val clusterIndex = randomInt(clusterHosts.size - 1)
+        val input = randomHttpInput(
+                scheme = clusterHosts[clusterIndex].schemeName,
+                host = clusterHosts[clusterIndex].hostName,
+                port = clusterHosts[clusterIndex].port,
+                path = "_cat/plugins",
```

Comment:
I changed the path from `_cluster/health` to `_cat/plugins` ( Here is the [code](https://github.com/ann3431/alerting/blob/430d718f96803ceec00b5fb50e664300fde6fa21/alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt#L591) in the original PR)